modules/mobile/app/controllers/mobile/v0/disability_rating_controller.rb

```diff
@@ -3,16 +3,35 @@
 module Mobile
   module V0
     class DisabilityRatingController < ApplicationController
-      before_action { authorize :evss, :access? }
+      before_action do
+        if Flipper.enabled?(:mobile_lighthouse_disability_ratings, @current_user)
+          authorize :lighthouse, :access?
+        else
+          authorize :evss, :access?
+        end
+      end
```

Comment:
This is just a nitpicky style preference, but can we add a blank line before the index method?

Reply:
no problem. rubymine likes to add newlines that aren't real lines to show annotations like who wrote it which is always confusing.